backend/dataall/modules/dataset_sharing/db/share_object_models.py

```diff
@@ -58,3 +58,11 @@ class ShareObjectItem(Base):
     S3AccessPointName = Column(String, nullable=True)
     status = Column(String, nullable=False, default=ShareItemStatus.PendingApproval.value)
     action = Column(String, nullable=True)
+
+
+class Catalog:
```

Comment:
I am not sure we will need this class if we work directly in the `lf_share_manager`. If it stays, I suggest we move it out of the `db` models and define it where it is used. I would also rename it to `GlueDatabase`

Reply:
Removed this class as not needed anymore